src/Interpreters/ExpressionAnalyzer.cpp

```diff
@@ -598,7 +597,7 @@ void ExpressionAnalyzer::makeAggregateDescriptions(ActionsDAGPtr & actions, Aggr
     }
 }
 
-void makeWindowDescriptionFromAST(const Context & context,
+void ExpressionAnalyzer::makeWindowDescriptionFromAST(const Context & context_,
```

Comment:
You can remove the first parameter because it's a member function now.

Reply:
It is still static member and we can't easily make that because basically lots of code for that class is made through static functions in corresponding cpp file. And same in two other cases in `InterpreterSelectQuery.cpp`.